retry/retry_test.go

```diff
@@ -232,6 +233,21 @@ func TestRetryModes(t *testing.T) {
 				nonIdempotentOperation: false,
 			},
 		},
+		{
+			err: xerrors.Retryable(
```

Comment:
Will not work - https://github.com/grpc/grpc-go/blob/v1.47.0/rpc_util.go#L842
Retry got status.Error without this error in nesting